src/widgets/chart.rs

```diff
@@ -226,6 +226,8 @@ pub struct Chart<'a> {
     style: Style,
     /// Constraints used to determine whether the legend should be shown or not
     hidden_legend_constraints: (Constraint, Constraint),
+    /// The position of a legend
+    legend_position: Corner,
```

Comment:
Would we ever want to put a legend say centered bottom, or vertically centered right? If so, what do we need as a positioning enum here?

Reply:
An alternative could be to draw the legend at certain coordinates, within the graph area, and provide default constructor functions for each of the four corners.

Reply:
That's not a bad idea - what would the configuration of the widget for that look like?